apps/prairielearn/src/pages/instructorAssessmentSettings/instructorAssessmentSettings.js

```diff
@@ -4,7 +4,7 @@ import * as async from 'async';
 import * as express from 'express';
 import * as path from 'path';
 const debug = require('debug')('prairielearn:' + path.basename(__filename, '.js'));
-import QR from 'qrcode-svg';
+const QR = require('qrcode-svg');
```

Comment:
Why is this a const but the detectMocha one is an import?

Reply:
This is a `.js` file; the other is a `.ts` file. `import ... = require(...)` is TS-specific syntax.